services/app-web/package.json

```diff
@@ -109,6 +110,7 @@
         "sass": "^1.49.11",
         "typescript": "^4.4.4",
         "url-parse": "^1.5.2",
+        "use-deep-compare-effect": "^1.8.1",
```

Comment:
Is this being used somewhere?

Reply:
Yes! `@apollo/explorer` requires `use-deep-compare-effect` to also be installed.